web-registry/src/pages/Dashboard/MyEcocredits/hooks/useOpenTakeModal.tsx

```diff
@@ -20,13 +20,14 @@ const useOpenTakeModal = ({
 }: Props): ReturnType => {
   const openTakeModal = useCallback(
     (rowIndex: number): void => {
-      const selectedBasketDenom =
-        basketsWithClasses?.[rowIndex]?.basket?.basketDenom;
```

Comment:
Instead of looking into the user basket tokens we were looking into all existing baskets with the selected row index. Which did not work as soon as the user has 2 baskets.